addons/sale_stock/static/src/sale_stock_forecasted/forecasted_details.xml

```diff
@@ -2,24 +2,33 @@
 <templates id="template" xml:space="preserve">
     <t t-name="sale_stock.ForecastedDetails" t-inherit="stock.ForecastedDetails" t-inherit-mode="extension">
         <xpath expr="//tr[@name='draft_picking_out']" position="after">
-            <tr t-if="props.docs.draft_sale_qty" name="draft_so_out" t-attf-class="#{props.docs.draft_sale_orders_matched and 'table-info'}">
-                <td colspan="2">Quotations</td>
-                <td t-out="_formatFloat(-props.docs.draft_sale_qty)" class="text-end"/>
+            <tr t-if="currentProduct.draft_sale_qty.out" name="draft_so_out" t-attf-class="#{currentProduct.draft_sale_orders_matched and 'table-info'} #{multipleProducts and 'collapse show' or ''} collapseGroup_#{line.product.id}">
+                <td/>
                 <td>
-                    <t t-foreach="props.docs.draft_sale_orders" t-as="sale_order" t-key="sale_order_index">
-                        <t t-if="sale_order_index > 0"> | </t>
-                        <a t-attf-href="#" t-out="sale_order.name"
-                            class="fw-bold"
-                            t-on-click.prevent="() => this.props.openView('sale.order', 'form', sale_order.id)"/>
-                    </t>
+                <t t-set="draft_so_out_count" t-value="currentProduct.draft_sale_orders.length"/>
+                <a t-if="draft_so_out_count == 1" href="#"
+                    t-on-click.prevent="() => this.props.openView('sale.order', 'form', currentProduct.draft_sale_orders[0].id)">
+                    1 Quotation</a>
+                <a t-else="" href="#"
+                    t-on-click.prevent="() => this.props.openView('sale.order', 'list', false, [['id', 'in', currentProduct.draft_sale_orders.map((so) => so.id)]])">
+                    <t t-out="draft_so_out_count"/> Quotations</a>
                 </td>
+                <td class="text-end">
+                    <span class="text-danger">
+                        <t t-out="_formatFloat(-currentProduct.draft_sale_qty.out)"/> <t t-out="line.uom_id.display_name"/>
+                    </span>
+                </td>
+                <td colspan="3"/>
             </tr>
         </xpath>
         <xpath expr="//td[@name='usedby_cell']" position="inside">
-            <span t-if="line.move_out and line.move_out.picking_id and line.move_out.picking_id.sale_id">
-                | <span t-out="_formatMonetary(line.move_out.picking_id.sale_id.amount_untaxed, line.move_out.picking_id.sale_id.currency_id.id)" class="fw-bold"/>
-                | <span t-out="line.move_out.picking_id.sale_id.partner_id.display_name"/>
-            </span>
+            <span t-if="line.move_out and line.move_out.picking_id and line.move_out.picking_id.sale_id" t-out="' - ' + line.move_out.picking_id.sale_id.partner_id.display_name"/>
+            <t t-if="line.reservation and (line.document_out._name != line.reservation._name or line.document_out.id != line.reservation.id)">
```

Comment:
picky but it would be great to use the model name directly instead of _name. That way we understand directly what the code is doing (like upper)